### DISCUSSION TITLE
"(MISSING)" string in Jenkins log when pulling the VM

### DISCUSSION THREAD

**Discussion by pavellety** [cirruslabs/cirrus-cli#630] [Q&A]
Hi, we are using cirrus CLI in Jenkins jobs. When the VM is pulled from the registry(Harbor) we see strange lines in the log

This is output of `cirrus run`
```
10:43:36
10:43:38  
10:43:38  pulling manifest...
10:43:38  
10:43:38  pulling disk (18.7 GB compressed)...
10:43:38  
10:43:38  0%!
10:43:38  (MISSING)
10:43:45  0%!
10:43:45  (MISSING)
10:43:51  3%!
10:43:51  (MISSING)
```

when we run tart directly `tart pull ....` the output is clear

```
11:43:36
11:43:38  
11:43:38  pulling manifest...
11:43:38  
11:43:38  pulling disk (18.7 GB compressed)...
11:43:38  
11:43:38  0%!
11:43:45  0%!
11:43:51  3%!
```

I know this may depends on my env settings and it's hard to debug. My question is only if you see any issue for the first sight?

Thanks,
Pavel

**Comment by edigaryev** [comment]
Converted into issue: https://github.com/cirruslabs/cirrus-cli/issues/631.
